examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -76,3 +76,14 @@
 # The estimator can now run on any Array API compliant libraries such as
 # `CuPy <https://docs.cupy.dev/en/stable/overview.html>`__, a GPU-accelerated array
 # library. For details, see the :ref:`User Guide <array_api>`.
+
+# %%
+# Improved efficiency of many estimators
+# --------------------------------------
+# In version 1.1 the efficiency of many estimators relying on the computation of
+# pairwise distances was greatly improved for float64 dense input. In version 1.2,
+# the efficiency of these estimators was further improved for all combinations of
```

Comment:
```suggestion
# the efficiency of these estimators were further improved for all combinations of
```

Reply:
I'm not sure about that, it's the efficiency that was improved, not the estimators.